cirrus/server/cirrus/main.py

```diff
@@ -221,14 +247,16 @@ async def compute_features(request_data: FeatureRequest):
     await record_metrics(
         enrolled_partial_configuration=enrolled_partial_configuration,
         client_id=request_data.client_id,
+        nimbus_preview_flag=nimbus_preview or False,
     )
 
     return client_feature_configuration
 
 
 async def fetch_schedule_recipes() -> None:
     try:
-        app.state.remote_setting.fetch_recipes()
+        app.state.remote_setting_live.fetch_recipes()
+        app.state.remote_setting_preview.fetch_recipes()
```

Comment:
Ah you know what I just realized, if we put these in the same exception block, then if live throws, preview will never be refreshed.  Might wanna split these into separate try blocks and add tests to ensure that if one throws the other still fetches.

Reply:
yup good point